content/configuration/external-configs.md

```diff
@@ -4,10 +4,36 @@ sort: 3
 contributors:
   - sokra
   - skipjack
+  - tarang9211
 ---
 
-...
+Webpack lets you define you configuration files in any language. The list of supported file extensions can be found at the [node-interpret](https://www.npmjs.com/package/interpret) package. Webpack will still interpret your file as pure JS!
 
-?> possible extensions, e.g. `.babel.js`
+For example if you use **coffeescript**, your file would be as follows:
 
-?> loader settings, e.g. `sassLoader`, `eslint` sections
\ No newline at end of file
+**webpack.config.coffee.js**
+
+```javascript
+HtmlWebpackPlugin = require('html-webpack-plugin')
+#installed via npm
+webpack = require('webpack')
+#to access built-in plugins
```

Comment:
I think, we can omit those two comments as they are used throughout the documentation.